src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.test.tsx

```diff
@@ -0,0 +1,346 @@
+
+interface ComponentsProps {
+  activeFilters?: FinancialAccountTransactionFilters;
+  searchTerm?: string;
+  router?: object;
+}
+
+const Components = ({
+  searchTerm = defaultSearchTerm,
+  activeFilters = defaultActiveFilters,
+  router = defaultRouter,
+}: ComponentsProps) => (
+  <I18nextProvider i18n={i18n}>
+    <LocalizationProvider dateAdapter={AdapterLuxon}>
+      <SnackbarProvider>
+        <ThemeProvider theme={theme}>
+          <TestRouter router={router}>
+            <GqlMockedProvider<{
+              FinancialAccount: FinancialAccountQuery;
+              FinancialAccountEntries: FinancialAccountEntriesQuery;
+            }>
+              mocks={{
+                FinancialAccount: defaultFinancialAccount,
+                FinancialAccountEntries: financialAccountEntriesMock,
+              }}
+              onCall={mutationSpy}
+            >
+              <FinancialAccountsWrapper>
+                <FinancialAccountContext.Provider
+                  value={
+                    {
+                      accountListId,
+                      financialAccountId,
+                      financialAccountsQuery: {
+                        data: defaultFinancialAccount,
+                        loading: false,
+                      },
+                      hasActiveFilters: false,
+                      setActiveFilters,
+                      setPanelOpen,
+                      searchTerm,
+                      activeFilters,
+                    } as unknown as FinancialAccountType
+                  }
+                >
+                  <AccountTransactions />
+                </FinancialAccountContext.Provider>
+              </FinancialAccountsWrapper>
+            </GqlMockedProvider>
+          </TestRouter>
+        </ThemeProvider>
+      </SnackbarProvider>
+    </LocalizationProvider>
+  </I18nextProvider>
+);
+
+describe('Financial Account Transactions', () => {
+  describe('Resetting filters', () => {
+    it('should reset the activeFilters with the filters from the url filters', () => {
+      render(
+        <Components
+          activeFilters={{ categoryId: 'test123' }}
+          router={{
+            ...defaultRouter,
+            query: {
+              ...defaultRouter.query,
+              filters: '{"categoryId":"newCategoryId"}',
+            },
+          }}
+        />,
+      );
+
+      expect(setActiveFilters).not.toHaveBeenCalledWith({
+        dateRange: {
+          min: defaultStartDate,
+          max: defaultEndDate,
+        },
+      });
+
+      expect(setActiveFilters).toHaveBeenCalledWith({
+        categoryId: 'newCategoryId',
+      });
+      expect(setPanelOpen).toHaveBeenCalledWith(Panel.Filters);
+    });
+
+    it('should set filters to default date range if no activeFilters or url filters', () => {
+      render(<Components />);
+
+      expect(setActiveFilters).toHaveBeenCalledWith({
+        dateRange: {
+          min: defaultStartDate,
+          max: defaultEndDate,
```

Comment:
Instead of using the exported default start and end date, I think I'd prefer calculating what we expect the start date and end date to be based on a today of January 1, 2020 (mocked in `setup.ts`). That way we're also testing that `defaultStartDate`/`defaultStartEnd` date are calculated correctly. But feel free to disagree.